apps/native/app/src/screens/document-detail/document-detail.tsx

```diff
@@ -274,7 +274,7 @@ export const DocumentDetailScreen: NavigationFunctionComponent<{
         },
       },
     })
-  }, [Document.id, Document.opened])
+  }, [Document.id])
```

Comment:
For some reason Document.opened is changing to `true` and then to `null` so this is triggered three times in a row, resulting in decreasing the unreadCount twice instead of once, so that is the reason I removed it from the dependency array. Works now as expected.